spec/helpers/application_helper_spec.rb

```diff
@@ -1352,4 +1352,47 @@ def controller
       expect(calculate_toolbars).to include("center_tb" => "container_projects_center")
     end
   end
+
+  describe '#provider_paused?' do
```

Comment:
So there's test coverage for the new method.

Do you think that is sufficient for such a big PR?

Reply:
As the size of the PR comes from cut&paste, I don't think this is such a big PR...